src/main/java/org/threadly/litesockets/NoThreadSocketExecuter.java

```diff
@@ -226,7 +214,6 @@ public void run() {
     }
     clients.clear();
     servers.clear();
-    dogCache.cleanAll();
```

Comment:
Right now there is no call to immediately purge the cache.  Does this seem important?  It will still cleanup as watchdogs finish.  I could easily add it before the 4.0.0 release.  If you do want this, don't merge this PR yet, give me your thoughts (if I had thought of it earlier I would have added it from the start, so maybe it's worth adding)